client/tests/smokeTest.test.js

```diff
@@ -98,7 +98,7 @@ describe('smoke test', () => {
         if (browser) await browser.close();
     }, 60000);
 
-    it('loads various pages without crashing', async () => {
+    it.skip('loads various pages without crashing', async () => {
```

Comment:
I assume this was related to the flaky test?  I think this was fixed and no longer requires a skip.

Reply:
Yes. Good catch